torch/_dynamo/variables/torch.py

```diff
@@ -249,7 +249,7 @@ def call_function(
             torch.cpu.amp.autocast,
         ):
             return AutocastModeVariable.create(self.value, args, kwargs)
-        elif self.value in (
+        elif 0 and self.value in (  # TODO: pearu hack
```

Comment:
Undo this change.

Reply:
yeah, this was just to see if it fixed the test run